src/api/spec/support/shared_examples/features/beta/user_tab.rb

```diff
@@ -148,6 +154,8 @@
     end
 
     scenario 'Add non existent group' do
+      skip_on_mobile
```

Comment:
We can do the same here as in the examples with `Add User`, just switching `Add User` to `Add Group` and it should work.

Reply:
On it.

Reply:
Done. I fixed the commit description again. Please take a look.

Reply:
This part of the commit message isn't true anymore: `Or skip the test in mobile.` It should be removed.

Reply:
Done!